lib/generators/install/templates/cable.yml.erb

```diff
@@ -0,0 +1,14 @@
+# production: &production
+#   :url: redis://localhost:6379
+#   :host: localhost
+#   :port: 6379
+#   :timeout: 1
+
+development: &development
+  :url: redis://localhost:6379
+  :host: localhost
+  :port: 6379
+  :timeout: 1
+  :inline: true

```

Comment:
It's a direct copy from the README, though the configuration section (and installation instructions in general) in the README will need to be rewritten after this commit.